ci-operator/step-registry/cucushift/installer/rehearse/gcp/ipi/user-labels-tags/provision/cucushift-installer-rehearse-gcp-ipi-user-labels-tags-provision-chain.yaml

```diff
@@ -0,0 +1,16 @@
+chain:
+  as: cucushift-installer-rehearse-gcp-ipi-user-labels-tags-provision
+  steps: 
+    - ref: gcp-provision-user-tags
+    - chain: ipi-conf-gcp
+    - ref: ipi-conf-gcp-user-labels-tags
+    - ref: ipi-conf-credentials-mode
```

Comment:
why `ipi-conf-credentials-mode` is invoked and no value is assigned?

Reply:
I saw the env is set in https://github.com/openshift/release/pull/42819/files#diff-a53d8367e2f8a5c3436a55ddde4ab4c86c6ea9797fd30ccd08c4d330a198d098R10, ignore my above comment.